### PR TITLE
added COD rivers functionality

### PR DESCRIPTION
**Pull request by SteveP2000**
Does not include validation as source contains limited schema information - e.g. river names are not included in dataset.

### CODE REVIEW

File: src/transform/rivers.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    # Change CRS
+    df_rivers = df_rivers[df_rivers['geometry'].notna()]
+    df_rivers = df_rivers.to_crs(config['constants']['crs'])
+    # Make columns needed for validation
+    df_rivers['geometry_type'] = df_rivers['geometry'].apply(lambda x: x.geom_type)
+    df_rivers['crs'] = df_rivers.crs
+
+    # TODO fix validation
+    # Validate
+    # validate(instance=df_rivers.to_dict('list'), schema=parse_yaml(schema_filename))
```
Comment (turnerm) [inline src/transform/rivers.py:28]: Need to add `from jsonschema import validate` to this file

File: schemas/rivers_affected_area_py.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+required:
+    - name_en
```
Comment (turnerm) [inline schemas/rivers_affected_area_py.yml:2]: Do we need this for rivers? (There's no schema defined in the summary of internal artefacts)

File: schemas/rivers_affected_area_py.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+properties:
+    geometry_type:
+        items:
+            enum:
+                - LineString
```
Comment (turnerm) [inline schemas/rivers_affected_area_py.yml:7]: Should also include `MultiLineString`

File: schemas/rivers_affected_area_py.yml (hunk 2 above)
Comment (SteveP2000) [reply schemas/rivers_affected_area_py.yml:2]: No schema defined. Removed from code

File: schemas/rivers_affected_area_py.yml (hunk 3 above)
Comment (SteveP2000) [reply schemas/rivers_affected_area_py.yml:7]: Added to code

**Pull request closed without merging**